index.js

```diff
@@ -235,7 +235,7 @@ class Client {
     /**
      * Get series and episode updates since a given unix timestamp
      *
-     * http://www.thetvdb.com/wiki/index.php?title=API:Update_Records
+     * http://thetvdb.com/wiki/index.php?title=API:Updates
```

Comment:
please can the url be `http://www.thetvdb.com/wiki/index.php?title=API:Updates` (added www.) to match other urls